roborazzi/src/main/java/com/github/takahirom/roborazzi/capture.kt

```diff
@@ -285,7 +285,8 @@ data class RoborazziOptions(
   }
 
   data class RecordOptions(
-    val resizeScale: Double = 1.0
+    val resizeScale: Double = 1.0,
+    val fullScreen: Boolean = true
```

Comment:
I believe adding the parameter to RecordOptions is the right choice! However, the name "fullscreen" might be a bit confusing. Users could potentially think it refers to a navigation bar or status bar. I think the name "applyDeviceCrop: Boolean = true" is a better alternative and should be clearer for users.
While "fullscreen" is useful for other features and might be a good name, do you have any suggestions for features that would work well with fullscreen?

Reply:
Done.  I think eventually it should be true for full screen tests, and defaulted to true.  But for now, changing to false until it works end to end.